src/components/Footer.tsx

```diff
@@ -0,0 +1,44 @@
+import { FC } from 'react';
+import { Todo } from '../types/Todo';
+import { TodosFilter } from './TodosFilter';
+
+type Props = {
+  todos: Todo[],
+  onTodoDelete: (todoIds: number[]) => void;
+};
+
+export const Footer: FC<Props> = ({ todos, onTodoDelete }) => {
+  const completedTodos = todos.filter(todo => todo.completed);
```

Comment:
Consider using a useMemo hook to memoize the filtered list of completed todos and the active todos count.